docs/fiddles/native-ui/drag-and-drop/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Drag and drop files</title>
+  </head>
+
+  <body>
+    <div class="section-wrapper">
+      <h1>Drag and drop files</h1>
+
+      <h3>
+        Electron supports dragging files and content out from web content into
+        the operating system's world.
+      </h3>
+
+      <p>
+        Open the
+        <a href="https://electronjs.org/docs/tutorial/native-file-drag-drop"

```

Comment:
External links in `electron-api-demos` have an event listener attached that uses Electron's `shell` API to open the link in an external browser.
```js
(link) => {
  const url = link.getAttribute('href')
  if (url.indexOf('http') === 0) {
    link.addEventListener('click', (e) => {
      e.preventDefault()
      shell.openExternal(url)
    })
  }
}
```

Without a similar listener in the Fiddle, this link opens in the same Electron window.

Reply:
added to renderer, is there a way to add this globally?